VimeoNetworking/Sources/Models/VIMLive.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Foundation
+
+/// The streaming status of a live video.
+///
+/// - unavailable: the RTMP link is visible but not yet able to receive the stream.
+/// - pending: Vimeo is working on setting up the connection.
+/// - ready: the RTMP's URL is ready to receive video content.
+/// - streamingPreview: the stream is in a "preview" state. It will be accessible to the public when you transition to "streaming".
+/// - streaming: The stream is open and receiving content.
+/// - streamingError: The stream has been terminated by Vimeo.
+/// - done: The stream has been ended intentionally by the end-user.
+public enum LiveStreamingStatus: String
+{
+    case unavailable = "unavailable"
+    case pending = "pending"
+    case ready = "ready"
+    case streamingPreview = "streaming_preview"
+    case streaming = "streaming"
+    case streamingError = "streaming_error"
+    case done = "done"
+}
+
+/// An object that represents the `live` field in
+/// a `clip` response.
+public class VIMLive: VIMModelObject
+{
+    /// An RTMP link used to host a live stream.
+    public private(set) var link: String?
+    
+    /// A token for streaming.
+    public private(set) var key: String?
+    
+    /// The timestamp that the stream is active.
+    public private(set) var activeTime: NSDate?
+    
+    /// The timestamp that the stream is over.
+    public private(set) var endedTime: NSDate?
+    
+    /// The timestamp that the live video is
+    /// archived.
+    public private(set) var archivedTime: NSDate?
+    
+    /**
+        The status of the live video in string.
+     
+        - Note:
+        Technically, this property should not be used to
+        check the status of a live video. Use
+        `liveStreamingStatus` instead for easy checking.
+     */
+    public private(set) var status: String?
+    {
+        didSet
```

Comment:
This is a bit confusing.  We shouldn't be using the didSet method of status to set the value of liveStreamingStatus.  Instead, liveStreamingStatus should be a computed variable that returns LiveStreamingStatus(rawValue: self.status).